web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -422,6 +423,11 @@ private QueryData validateQuery(String queryLogicName, MultivaluedMap<String,Str
         if (params != null) {
             for (Parameter pm : QueryUtil.parseParameters(params)) {
                 if (!queryParameters.containsKey(pm.getParameterName())) {
+                    if (pm.getParameterName().equals(INVALID_PAGESIZE)) {
+                        log.error("Invalid parameter: page.size");
```

Comment:
In the log and/or the error message, maybe it's worth specifying the name of the correct parameter to use.

Reply:
Can do, I'll update this on next commit.